opentargets.json

```diff
@@ -1728,6 +1820,10 @@
         "OTAR033"
       ]
     },
+    "projectDescription": {
```

Comment:
This field differs from `studyOverview` in that this refers to the broader scope of the project rather than to the description of the study where the evidence was observed, right?

Reply:
Yes, those two are different. From one project (`OTAR####`) can source multiple studies with their corresponding studyOverview description.